tests/test_regression.py

```diff
@@ -295,7 +299,7 @@ def collation_cb(a, b):
 
     def test_RecursiveCursorUse(self):
         # note: modified test slighty since we actually just handle this fine.
-        con = monetdbe.connect(":memory:")
+        con = self.con
```

Comment:
why is this required?

Reply:
In the setUp a connection object is already created. There is no reason for this test not to use that. And the previous code was either leaking connection objects or it trusted on the GC to timely destroy the connection object. Unless you take explicit control over the GC, this can become problematic once you try to open a connection to a _different_ db because monetdbe does not currently allow concurrent connections to different db's.

Reply:
ok good, just curious what your motivations were here.